Sources/DualCameraKit/DualCameraEnvironment.swift

```diff
@@ -4,8 +4,21 @@ import UIKit
 /// A value-type container for default services,
 /// allowing dependency injection with safe, testable defaults.
 /// Inspired by https://www.pointfree.co/episodes/ep16-dependency-injection-made-easy
+///  Trying this out for now. It still feels kinda weird and so
+///  at some point it might  be worth  moving it to a more formal DI.
 public struct DualCameraEnvironment: Sendable {
     public var mediaLibraryService: MediaLibraryService = .live()
+    public var dualCameraController: DualCameraControlling = Self.getDefaultCameraController()
+    
+    
+    @MainActor
```

Comment:
this is triggering a warning re: mainactor isolation. followup issue here https://github.com/Liampronan/DualCameraKit/issues/39